rules/Php80/Rector/NullsafeMethodCall/CleanupUnneededNullsafeOperatorRector.php

```diff
@@ -0,0 +1,105 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Rector\Php80\Rector\NullsafeMethodCall;
+
+use PhpParser\Node;
+use PhpParser\Node\Expr\MethodCall;
+use PhpParser\Node\Expr\NullsafeMethodCall;
+use Rector\Core\Rector\AbstractRector;
+use Rector\Core\ValueObject\PhpVersionFeature;
+use Rector\TypeDeclaration\TypeAnalyzer\ReturnStrictTypeAnalyzer;
+use Rector\VersionBonding\Contract\MinPhpVersionInterface;
+use Symplify\RuleDocGenerator\ValueObject\CodeSample\CodeSample;
+use Symplify\RuleDocGenerator\ValueObject\RuleDefinition;
+
+/**
+ * @see https://wiki.php.net/rfc/nullsafe_operator
+ *
+ * @see \Rector\Tests\Php80\Rector\NullsafeMethodCall\CleanupUnneededNullsafeOperatorRector\CleanupUnneededNullsafeOperatorRectorTest
+ */
+final class CleanupUnneededNullsafeOperatorRector extends AbstractRector implements MinPhpVersionInterface
+{
+    public function __construct(
+        private readonly ReturnStrictTypeAnalyzer $returnStrictTypeAnalyzer,
+    )
+    {
+    }
+
+    public function getRuleDefinition(): RuleDefinition
+    {
+        return new RuleDefinition(
+            'Cleanup unneeded nullsafe operator',
+            [
+                new CodeSample(
+                    <<<'CODE_SAMPLE'
+class HelloWorld {
+    public function getString(): string
+    {
+         return 'hello world';
+    }
+}
+
+public function get(): HelloWorld
+{
+     return new HelloWorld();
+}
+
+echo $this->get()?->getHelloWorld();
+CODE_SAMPLE
+                    ,
+                    <<<'CODE_SAMPLE'
+class HelloWorld {
+    public function getString(): string
+    {
+         return 'hello world';
+    }
+}
+
+public function get(): HelloWorld
+{
+     return new HelloWorld();
+}
+
+echo $this->get()->getHelloWorld();
+CODE_SAMPLE
+                ),
+            ]
+        );
+    }
+
+    /**
+     * @return array<class-string<Node>>
+     */
+    public function getNodeTypes(): array
+    {
+        return [NullsafeMethodCall::class];
+    }
+
+    /**
+     * @param NullsafeMethodCall $node
+     */
+    public function refactor(Node $node): ?Node
+    {
+        if (!$node->var instanceof MethodCall) {
+            return null;
+        }
+
+        $returnNode = $this->returnStrictTypeAnalyzer->resolveMethodCallReturnNode($node->var);
+
+        if (null === $returnNode) {
+            return null;
+        }
+
+        // Remove not needed Nullsafe for method call.
+        $node = $node->var;
```

Comment:
I now remove `->getReplace();` as well is there a way to keep that maybe i should get parent call and get it from there?